dask_cloudprovider/__init__.py

```diff
@@ -42,3 +42,8 @@ def __getattr__(name):
             "DigitalOcean cluster managers must be imported from the digitalocean subpackage. "
             f"Please import dask_cloudprovider.digitalocean.{name}"
         )
+    if name in ["IBMCodeEngineCluster"]:
+        raise ImportError(
+            "IBM cluster managers must be imported from the ibm subpackage. "
+            f"Please import dask_cloudprovider.ibm.{name}"
+        )
```

Comment:
I wouldn't worry about this. These shims were added when we rearranged the package at one point. But given that nobody h as ever imported `IBMCodeEngineCluster` from this location I don't think we need it.